util/migdb-add-unit

```diff
@@ -0,0 +1,245 @@
+#!/bin/bash
+
+SCRIPT_PATH=$(dirname "${0}")
+# shellcheck source=/home/thomas/develop/balena.io/migrate/util/migdb-common
+source "${SCRIPT_PATH}/migdb-common"
+
+color ON
+
+if [ -z "$MIG_DB_DIR" ] ; then
+  MIG_DB_DIR=.
+fi
+
+#shellcheck disable=SC2034
```

Comment:
Are these intentionally unused variables?  I would be wary of disabling any shellcheck warnings, just because. So wondering about the rationale of doing it here (and in other places in the scripts
https://github.com/koalaman/shellcheck/wiki/SC2034

Reply:
These are variables that are used in migdb-common which I include. Unfortunately I have not sorted out the #shellcheck source= command properly (you mentioned above). This will be set right when I do.